ctp2_code/ai/diplomacy/sstateevent.cpp

```diff
@@ -701,7 +701,7 @@ STDEHANDLER(DefenseLevel_NextSStateEvent)
 	Diplomat & diplomat = Diplomat::GetDiplomat(playerId);
 	diplomat.ClearEffectiveRegardCache();
 
-	AiState state;
+	AiState thisState;
```

Comment:
Inconsistent naming previously it was called 'thisstate' with small 's'. I prefer this with capital 'S'. Still I prefer not to prefix it with 'the' or 'this'. In other places in the code 'the' is used.

Reply:
Agreed - this is an unecessary change and was motivated by misreading a different ld error on Fedora 32 when configured with --enable-debug due to int state being defined in both ldl.l and slic.l

I will revert this change.